README.md

```diff
@@ -1,6 +1,11 @@
 ![logo](tokbox-logo.png)
 
-# OpenTok Accelerator Archiving JavaScript<br/>Version 1.0.+
+[![Build Status](https://travis-ci.org/opentok/accelerator-archiving-js.svg?branch=master)](https://travis-ci.org/opentok/accelerator-core-js)
+[![GitHub release](https://img.shields.io/github/release/opentok/accelerator-archiving-js.svg)](./README.md)
+[![license](https://img.shields.io/github/license/opentok/accelerator-archiving-js.svg)](./.github/CONTRIBUTING.md)
+[![npm](https://img.shields.io/npm/v/opentok-archiving.svg)](https://www.npmjs.com/package/opentok-accelerator-core)
```

Comment:
@PanoramicRum I think it should be: https://www.npmjs.com/package/opentok-archiving